modules/samples-operator-overview.adoc

```diff
@@ -47,3 +47,20 @@ the following upon deletion:
 
 Upon deletion of the samples resource, the Samples Operator recreates the
 resource using the default configuration.
+
+[id="samples-operator-bootstrapped"]
+== Sample Operator's use of management state
+
+The Samples Operator is bootstrapped as `Managed` by default, which means that the Operator is actively managing its resources and keeping the component active in order to pull sample imagestreams and images from the registry.
```

Comment:
change

`pull sample imagestreams and images from the registry.`

to 

`install sample templates and imagestreams and ensure those sample imagestreams import their images.`